lib/relay/relay.dart

```diff
@@ -91,6 +94,34 @@ abstract class Relay {
     }
   }
 
+  /// Returns a list of all active subscriptions for this relay connection.
+  List<Subscription> getSubscriptions() {
+    return _subscriptions.values.toList();
+  }
```

Comment:
This would be more idiomatic as a getter instead:

```suggestion
  List<Subscription> get subscriptions => _subscriptions.values.toList();
```